lib/bindings/HTTPBinding.js

```diff
@@ -470,15 +470,26 @@ function returnCommands(req, res, next) {
     if (req.query && req.query.getCmd === '1') {
```

Comment:
CNR entry missing, mentioning the fixed issue.

Is there a twin PR with a similar fix also for IOTA-UL? Or in that IOTA this is ok?

Reply:
Iota UL https://github.com/telefonicaid/iotagent-json/issues/582#issuecomment-883229378 is returning a command in a a non json format, so IMHO there is no sense to allow return result in JSON

Reply:
Ok, application/json is not the right one in that case. However:

```
HTTP/1.1 200 OK
X-Powered-By: Express
Content-Type: text/html; charset=utf-8
Content-Length: 16
ETag: W/"10-qjxBJ6diMRAMIjnHPy48XmcadZE"
Date: Tue, 20 Jul 2021 10:21:28 GMT
Connection: keep-alive

disp3@close|s=36
```

should it be text/plain instead of text/html ?

Reply:
Fixed in independent PR https://github.com/telefonicaid/iotagent-ul/pull/507